3party/protobuf/CMakeLists.txt

```diff
@@ -66,6 +66,7 @@ endif()
 if (PLATFORM_WIN AND MSVC)
   set(
     SRC
+    ${SRC}
```

Comment:
I agree that they look ugly, but I suggest replacing them in a separate PR, as it will affect many files that are not directly related to this pull-request.